src/test/java/com/training/assignment/tests/AssignmentTest.java

```diff
@@ -1,13 +1,16 @@
 package com.training.assignment.tests;
 
 import com.training.assignments.Assignment;
+import com.training.assignments.Operations;
 import org.junit.jupiter.api.Test;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public class AssignmentTest extends BaseTest{
+public class AssignmentTest extends BaseTest {
     private Logger logger = LoggerFactory.getLogger(this.getClass());
+
     Assignment assignment = new Assignment();
+    Operations operation = new Operations();
```

Comment:
define assignment, operations as private.
Make sure all the variables defined are private